samples/subsys/usb/cdc_acm/sample.yaml

```diff
@@ -18,6 +18,9 @@ tests:
       - nrf52840dk/nrf52840
       - frdm_k64f
       - 96b_carbon/stm32f401xe
+      - mimxrt685_evk/mimxrt685s/cm33
+      - lpcxpresso55s69/lpc55s69/cpu0
+      - mimxrt1060_evk
```

Comment:
Please only one entry per controller type. I guess mimxrt1060_evk uses the same controller type as mimxrt685_evk/mimxrt685s/cm33?

Reply:
mimxrt1060_evk uses the ehci and the phy ip; lpc55s69 and mimxrt685s uses the same IP3511 and the phy ip. I will remove one later.